### PR TITLE
Work

### PR DESCRIPTION
**Pull request by jfurcht21**

### CODE REVIEW

File: src/main/java/Problem3/MovieAction.java [hunk 1]
```diff
@@ -22,5 +24,21 @@ public int getLateFeeInDollar() {
     @Override
     public int calcLateFees(int numOfDaysPastDue) {
         // homework
+        int lateFee = 0;
+        if(numOfDaysPastDue <= 0)
+        {
+            return 0;
+        }
+        if(numOfDaysPastDue > 0 && numOfDaysPastDue < 5)
```
Comment (jmkelly6) [inline src/main/java/Problem3/MovieAction.java:32]:
```suggestion
        int fee = lateFeePerDayInDollar * numOfDaysPastDue;
        return (numOfDaysPastDue >= 5) ? 2 * fee : fee;
```

File: src/test/java/Problem2Test.java [hunk 2]
```diff
@@ -7,13 +7,17 @@
     public void testRemoveDuplicates() {
         int[][] inputs = {
                 // DO NOT REMOVE OR CHANGE THE GIVEN TEST CASES!
-                {0, 0, 1, 1, 1, 2, 2, 4, 4, 6}
+                {0, 0, 1, 1, 1, 2, 2, 4, 4, 6},
                 // add more test cases
+                {1, 1, 1, 2, 2, 3, 4, 6, 9},
+                {0, 0, 1, 1, 2, 2, 3, 4, 6, 9}
```
Comment (jmkelly6) [inline src/test/java/Problem2Test.java:13]: Suggestion for future assignments: add edge case tests (ex. when an object is NULL or empty).

File: src/main/java/Problem2.java [hunk 3]
```diff
@@ -1,12 +1,52 @@
...
+            if (nums[i] != nums[i+1])
+            {
+                nums[unique] = nums[i];
+                unique++;
+            }
+
+        }
+        return unique;
+
+        /*int duplicateCounter = 0;
```
Comment (jmkelly6) [inline src/main/java/Problem2.java:37]: Don't forget to remove commented-out code

**Review by pdgetrf — APPROVED** [review]

**Review by pdgetrf — APPROVED** [review]
LGTM
